### DISCUSSION TITLE
integration in ESPhome

### DISCUSSION THREAD

**Discussion by pkkr** [ESPresense/ESPresense#233] [Ideas]
Hey, awesome project thanks for this. What are you thoughts about integration into ESPhome ?

**Comment by DTTerastar** [comment]
It's all in here: https://github.com/ESPresense/ESPresense/issues/73

**Reply by pkkr** [reply]
ops sry, must have overseen that. Thanks.

**Reply by DTTerastar** [reply]
No problem!
